cernopendata_client/searcher.py

```diff
@@ -219,11 +233,27 @@ def get_files_list(
                 else:
                     files_list_expanded.append(file_)
             files_list = files_list_expanded
-        if protocol == "http":
+    if protocol == "http":
+        if verbose:
+            files_list = [
+                (file_[0].replace(SERVER_ROOT_URI, server), file_[1], file_[2])
+                for file_ in files_list
+            ]
+        else:
             files_list = [
                 file_.replace(SERVER_ROOT_URI, server) for file_ in files_list
             ]
-        elif protocol == "https":
+    elif protocol == "https":
```

Comment:
Can protocol be `https`? I think we have removed it fully, so it is only `http` or `xrootd`.  It is `server` that sets up HTTP or HTTPS. So we can probably simplify this code.

Reply:
Yes, It can be if `--server` is using `HTTPS`.

Reply:
I mean do we even need to pass `protocol=https` to these functions in the searcher? For consistency, shouldn't we pass `protocol=http` and `server=https...` as we do in top-level CLI?  In this way, the Searcher expectations about server/protocol would fully comply with the CLI top-level notion of what server/protocol values might be... I'm wondering because inside Seacher we only use `protocol`, do no manipulation:

```console
$ git grep protocol cernopendata_client/searcher.py
cernopendata_client/searcher.py:167:    server=None, record_json=None, protocol=None, expand=None, verbose=N>
cernopendata_client/searcher.py:173:    :protocol: Protocol to be used in links [http,xrootd]
cernopendata_client/searcher.py:178:    :type protocol: str
cernopendata_client/searcher.py:208:    if protocol == "http":
cernopendata_client/searcher.py:213:    elif protocol == "https":
cernopendata_client/searcher.py:225:def get_file_info_remote(server, recid, protocol=None, filtered_files=No>
cernopendata_client/searcher.py:248:        if protocol == "http":
cernopendata_client/searcher.py:250:        elif protocol == "https":
```

The consistency between top-level CLI notion behind "protocol" and low-level Searcher notion behind "protocol", would avoid possible confusions later on.

Anyway, this could be also addressed later.